### PR TITLE
feat: add 1.4.1 contracts for OP Celestia Raspberry

### PR DESCRIPTION
**Pull request by ElvisKrop**
- relevant issue: https://github.com/safe-global/safe-deployments/issues/510

### CODE REVIEW

**Review by nlordell — APPROVED** [review]

**Comment by nlordell** [comment]
@ElvisKrop - changes look good. Please resolve merge conflicts for merging. Will likely conflict with #512.

**Comment by ElvisKrop** [comment]
@nlordell this one done, thanks!

File: src/assets/v1.4.1/compatibility_fallback_handler.json [hunk 1]
```diff
@@ -379,3 +380,4 @@
     }
   ]
 }
+
```
Comment (nlordell) [inline src/assets/v1.4.1/compatibility_fallback_handler.json:383]: Looks like an extraneous newline.

Comment (ElvisKrop) [reply src/assets/v1.4.1/compatibility_fallback_handler.json]: fixed [ee104ee](https://github.com/safe-global/safe-deployments/pull/511/commits/ee104ee40fb58ebb17d1f9a87b3afd51152f24b7)

**Review by nlordell — APPROVED** [review]

**Pull request merged**